package.json

```diff
@@ -47,6 +48,18 @@
     "webpack-dev-server": "2.7.1"
   },
   "devDependencies": {
+    "eslint": "^4.19.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-prettier": "^2.6.0",
+    "husky": "^0.14.3",
```

Comment:
Why do we need husky?

Reply:
https://github.com/AndelaOSP/Andela-Socials/pull/11 adds husky. I'll actually prefer this to be in a separate PR

Reply:
Since husky comes with git hooks and we would need to run prettier whenever some one is committing their code, it is needed for this PR.